backend/tests/test_google_verify.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase
```

Comment:
In my PR, I made a standardized factory for creating endpoint test classes. Once that's merged in, we need to convert this one as well.

Reply:
sure